src/main/java/kernitus/plugin/OldCombatMechanics/scheduler/SchedulerManager.java

```diff
@@ -0,0 +1,22 @@
+package kernitus.plugin.OldCombatMechanics.scheduler;
+
+public enum SchedulerManager {
+
+    INSTANCE;
+    private IScheduler scheduler;
+
+    public void loadPlatform() {
+        try {
+            Class<?> pluginMetaClass = Class.forName("io.papermc.paper.plugin.configuration.PluginMeta");
```

Comment:
Is there no public API to figure this out?

Reply:
Currently folia related api are judged through reflection, such as UniversalScheduler